lib/layouts/widgets/contact_avatar_group_widget.dart

```diff
@@ -38,311 +48,313 @@ class _ContactAvatarGroupWidgetState extends State<ContactAvatarGroupWidget> {
       );
     }
 
-    return Container(
-      width: widget.size,
-      height: widget.size,
-      child: participants.length > 1
-          ? ThemeSwitcher(
-              iOSSkin: Obx(
-                () {
-                  int maxAvatars = SettingsManager().settings.maxAvatarsInGroupWidget.value;
-                  return Stack(
-                    children: [
-                      Container(
-                        width: widget.size,
-                        height: widget.size,
-                        decoration: BoxDecoration(
-                          borderRadius: BorderRadius.circular(widget.size / 2),
-                          color: context.theme.accentColor.withOpacity(0.6),
+    return Obx(
+      () => Container(
+        width: widget.size,
+        height: widget.size,
+        child: participants.length > 1
+            ? ThemeSwitcher(
+                iOSSkin: Obx(
```

Comment:
I don't really think the nested obx is necessary here now that you have one surrounding the whole thing. I'd try removing it and make sure everything works as intended otherwise add it back I guess.

Reply:
Got it to work without